editor/src/clj/editor/game_object.clj

```diff
@@ -136,34 +136,40 @@
       (update :display-order (fn [display-order]
                                (vec (distinct (concat display-order (:display-order source-properties))))))))
 
-(defn resource-path-error [_node-id source-resource]
+(defn- resource-path-error [_node-id source-resource]
     (or (validation/prop-error :fatal _node-id :path validation/prop-nil? source-resource "Path")
         (validation/prop-error :fatal _node-id :path validation/prop-resource-not-exists? source-resource "Path")
         (validation/prop-error :fatal _node-id :script validation/prop-resource-not-component? source-resource "Path")))
 
-(g/defnk produce-component-build-targets [_node-id build-resource ddf-message pose resource-property-build-targets source-build-targets]
-  ;; Create a build-target for the referenced or embedded component. Also tag on
+(g/defnk produce-referenced-component-build-targets [_node-id source-resource ddf-message pose resource-property-build-targets source-build-targets]
+  ;; Create a build-target for the referenced component. Also tag on
```

Comment:
Split into `produce-referenced-component-build-targets` and `produce-embedded-component-build-targets`.